editor/src/core/workers/parser-printer/parser-printer-utils.spec.ts

```diff
@@ -52,18 +52,18 @@ describe('guaranteeUniqueUidsFromTopLevel', () => {
       jsxElement(
         'View',
         'root',
-        jsxAttributesFromMap({ 'data-uid': jsxAttributeValue('root', emptyComments) }),
+        jsxAttributesFromMap({ 'data-uid': jsExpressionValue('root', emptyComments, 'root') }),
```

Comment:
wait a minute! if the unique ID of root is root. the unique ID of the string setting data-uid to root cannot be root!! because these two uids share the same global uid space, right?

Reply:
Very true, so I would expect it to be update I think but the test in this case doesn't validate that, just the siblings.